instance/tests/integration/factories/instance.py

```diff
@@ -79,8 +79,8 @@ def create(cls, *args, **kwargs):  # pylint: disable=arguments-differ
     # or a release candidate tag will work.  We point both the edx-platform and the configuration
     # versions to the branch "integration" in our own forks.  These branches are based on the
     # corresponding openedx_release versions from upstream, but can contain custom modifications.
-    openedx_release = 'open-release/ironwood.2'
+    openedx_release = 'open-release/juniper.2'
     configuration_source_repo_url = 'https://github.com/open-craft/configuration.git'
-    configuration_version = 'integration-ironwood'
+    configuration_version = 'integration-juniper'
```

Comment:
@0x29a, you can drop this change from this PR if this is causing CI check failures. We can investigate and fix the issues and create a separate PR.

Reply:
@lgp171188, ah, okay, I'll do this. Thanks.

Reply:
@lgp171188 done.